front/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:buster
+
+RUN apt update
+RUN echo yes | apt upgrade
```

Comment:
Those two RUN can be  combined into one, thus saving the creation of an additional layer to the image 
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers


also, I reckon `update -y` than an orphaned echo yes haha

Reply:
yes indeed, was just skip optimisation. Will be fixed next coommit